src/projections.F90

```diff
@@ -1496,7 +1498,7 @@ subroutine point_plane(pt, p0, v1, v2, n, sol, n_sol, best_sol)
 
   ! Now post-process to get the closest one
   best_sol = 1
-  D0 = norm(p0(:, ind(1)) + sol(2, ind(1))*v1(:, ind(1)) + sol(3, ind(1))*v2(:, ind(1)))
+  D0 = norm(p0(:, ind(1)) + sol(2, ind(1))*v1(:, ind(1)) + sol(3, ind(1))*v2(:, ind(1)), 3)
```

Comment:
I'm assuming the missing parameter here is 3, please double check.

Reply:
I am having trouble figuring out where this `norm()` function is. Do you know where it references?

Reply:
The norm function is defined in `compute_curve.f90`. With 2008 we could replace this with `norm2`. Maybe in a separate PR.

Reply:
Right, I would prefer a separate PR if we want to update to modern intrinsics.

Reply:
Never mind I just did it here